Xamarin.Forms.Core/Internals/CommandExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Windows.Input;
+
+namespace Xamarin.Forms.Internals
+{
+	static class CommandExtensions
+	{
+		public static void Run(this ICommand command)
+		{
+			if (command?.CanExecute(null) == true)
+			{
```

Comment:
this is DRY pushed to the extreme. It's actually harder for me to think about what Run does than having to repeat those 2 lines, even at 20 different places.

Reply:
also `CanExecute == true`...

I know it's to resolve the confusion between bool and bool?, so in this case I wouldn't use the null conditional operator

```
if (command != null && command.CanExecute(null))
```

Reply:
I encountered errors a couple of times when the command was null or something like that.
So the idea came up to make such an extension, and I fix comments about checking for null :)